src/main/haskell/language-kore/src/Data/Kore/ASTPrettyPrint.hs

```diff
@@ -768,4 +768,4 @@ instance (MetaOrObject level, PrettyPrint (variable level))
     prettyPrint flags (FunctionalVariable v) =
         writeOneFieldStruct flags "FunctionalVariable" v
     prettyPrint flags (FunctionalHead h) =
-        writeOneFieldStruct flags "FunctionalHead" h
+        writeOneFieldStruct flags "FuncitonalHead" h
```

Comment:
My bad, this is me trying to revert a file and not knowing git.

Reply:
Same with all the other whitespace errors and .DS_Store

I could've sworn all those were left out... guess one made it in because i type * at some point.